app/packages/core/src/plugins/SchemaIO/components/OperatorExecutionButtonView.tsx

```diff
@@ -0,0 +1,159 @@
+import React from "react";
+import { MuiIconFont } from "@fiftyone/components";
+import { OperatorExecutionButton, usePanelEvent } from "@fiftyone/operators";
+import { usePanelId } from "@fiftyone/spaces";
+import { isNullish } from "@fiftyone/utilities";
+import { Box, ButtonProps, Typography } from "@mui/material";
+import { getColorByCode, getComponentProps, getDisabledColors } from "../utils";
+import { ViewPropsType } from "../utils/types";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import TooltipProvider from "./TooltipProvider";
+
+export default function ButtonView(props: ViewPropsType) {
```

Comment:
Is the filepath and the export function name divergence purposeful here? @ritch 

The filepath is: `app/packages/core/src/plugins/SchemaIO/components/OperatorExecutionButtonView.tsx`
But the export function name is regular `ButtonView(...)`

Reply:
Thanks for catching this, it should indeed match the file path!